src/palace/manager/api/circulation.py

```diff
@@ -636,19 +636,15 @@ def __init__(self, _db: Session, collection: Collection):
     def collection(self) -> Collection | None:
         return Collection.by_id(self._db, id=self.collection_id)
 
-    @classmethod
     def default_notification_email_address(
-        self, library_or_patron: Library | Patron, pin: str | None
-    ) -> str:
+        self, patron: Patron, pin: str | None
```

Comment:
There was a test to make sure that this could work with a library, but all the places its actually called pass a `patron` and the overdrive implementation only allows a `patron`. So I updated the hint on the base class and removed the test where its called with a library.